core/service/json_parse.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef IROHA_JSON_PARSE_H
+#define IROHA_JSON_PARSE_H
+
+#include <string>
+#include <map>
+#include <memory>
+#include <vector>
+
+namespace json_parse {
+
+    enum class Type {
+        BOOL,
+        STR,
+        INT,
+        FLOAT,
+        LIST,
+        DICT
+    };
+
+    struct Object {
+        Type type;
+        int integer;
+        std::string str;
+        bool boolean;
+        float floating;
+
+        Object(Type t,int i) :
+            type(t), integer(i)
+        {
+            if(t != Type::INT) throw "TypeError"; //WIP
+        }
+
+        Object(Type t,std::string s) :
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'Object::integer' is not initialized in the constructor.](https://www.codacy.com/app/soramitsu/iroha/file/4345446921/issues/source?bid=3712491&fileBranchId=3870372#l33)